content/300-guides/050-database/880-supabase.mdx

```diff
@@ -28,6 +28,28 @@ Many aspects of using Prisma with Supabase are just like using Prisma with any o
 - use [`db push`](/concepts/components/prisma-migrate/db-push) to push changes in your schema to Supabase
 - use [Prisma Client](/concepts/components/prisma-client) in your application to talk to the database server at Supabase
 
+## Specific considerations
+
+If your application is being deployed to a serverless environment, make sure that you follow our general guidance on [connection pooling](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#recommended-connection-pool-size-1). Supabase makes this easy by [providing connection pooling](https://supabase.com/docs/guides/integrations/prisma#connection-pooling-with-supabase) compatible with PgBouncer.
+
+In short, you will need to update your Prisma schema to include `url` as well as `direct_url` as discussed [here](https://www.prisma.io/docs/data-platform/data-proxy/prisma-cli-with-data-proxy#set-a-direct-database-connection-url-in-your-prisma-schema). In addition, you will need to add `pgbouncer=true` to the end of your `DATABASE_URL` string.
```

Comment:
I would turn this around. Default action is to use `pgbouncer=true`. And then if you also want to use Prisma CLI to e.g. migrate your database (and you probably want to), `directUrl` makes it possible to also define an unpooled connection string with `directUrl` that is used instead of the default `url` one.